src/Psl/ObjectOriented/get_class.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\ObjectOriented;
+
+use function get_class as php_get_class;
+
+/**
+ * Returns the name of the class of an object.
+ *
+ * Example:
+ *
+ *      ObjectOriented\get_class($vector);
+ *      => Str('Psl\Collection\Vector')
+ *
+ * @template T of object
+ *
+ * @param T $object
+ *
+ * @return class-string<T>
+ *
+ * @psalm-pure
+ */
+function get_class(object $object): string
+{
+    return php_get_class($object);
```

Comment:
What is the end-user's benifit of using these kind of functions instead of the internal php function?
Is it just for the sake of providing everything?

I've seen these functions in other places in PSL as well, but in my code I avoided these and used the php internal once instead.

Reply:
better typed. guarantee to have the same behavior as other user land code, and no type juggling.

In this case specifically, no benefits, but to be consistent and provide everything.